R/model-tools.R

```diff
@@ -31,43 +31,29 @@
 #' enw_formula_as_data_list(prefix = "missing")
 enw_formula_as_data_list <- function(formula, prefix,
                                      drop_intercept = FALSE) {
-  paste_lab <- function(string, lab = prefix) {
-    paste0(lab, "_", string)
-  }
+  data <- within(list(), {
+    fintercept <- fnrow <- fnindex <- fncol <- rncol <- 0
+    fdesign <- findex <- rdesign <- numeric()
+  })
   if (!missing(formula)) {
     if (!inherits(formula, "enw_formula")) {
-      stop(
-        "formula must be an object of class enw_formula as produced using
-        enw_formula"
-      )
+      stop("
+        formula must be an object of class enw_formula as produced using
+        enw_formula")
```

Comment:
Spacings a bit inconsistent here with how it works elsewhere in the library - did the linter suggest this? Bit curious to see that message if so.